### PR TITLE
Improve layout of help pages

### LINKED ISSUE #101: Fix Help Page Layout
The presence of the relevant law on various help page sections has shifted the focus somewhat, we need to remove it but provide a link/description to the law for those who are interested. 

Consider placing the name of the law, in smaller text, below each relevant section with a 'ver mas' button to expand the first paragraph of text and a 'ver lea' to link to the full text.

### PR DESCRIPTION
**Pull request by zarino**
Fixes #101.

* Removes `text-align: justify`
* Reduces font-size and margins for legal section headings
* Hides all but the first paragraph of legal text, behind a "show more" link

screen shot 2015-01-19 at 14 57 59

### CODE REVIEW

File: assets/javascripts/legal-summaries.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+(function($) {
+
+  var summarise = function(){
```
Comment (stevenday) [inline assets/javascripts/legal-summaries.js]: Very minor thing, but if you name this function, e.g.
```javascript
var summarise = function summarise() {
```

It gets a name when it's debugged, which is nice.

File: assets/javascripts/legal-summaries.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    var $div = $(this);
+    var $firstP = $('p:first-of-type', $div);
+    var $extraElements = $firstP.nextAll();
+
+    if($extraElements.length > 0){
+
+      if($('html').attr('lang') == 'es'){
+        var readMore = '(Ver mas)';
+      } else {
+        var readMore = '(Read more)';
```
Comment (stevenday) [inline assets/javascripts/legal-summaries.js]: My JSLint is unhappy about this definition inside an `if`. Could you switch this to:
```javascript
var readMore = '(Ver mas)';
if($('html').attr('lang') == 'en'){
       readMore = '(Read more)';
}
(The logic then matches the site's language preferences too)

**Comment by stevenday** [comment]
Minor JS quibbles aside, :+1:

**Comment by zarino** [comment]
@stevenday – Done!

**Comment by lizconlan** [comment]
Looks much nicer! :smile:

**Pull request merged**